src/main/java/br/unicamp/cst/bindings/rosjava/RosServiceClientCodelet.java

```diff
@@ -93,22 +93,22 @@ public void onFailure(RemoteException e) {
 
 	@Override
 	public synchronized void start() {
-		startRosNode();
+                startRosNode();
 		super.start();
 	}
 
 	@Override
 	public synchronized void stop() {
-		stopRosNode();
+                stopRosNode();
 		super.stop();
 	}
 
 	private void startRosNode() {
-		nodeMainExecutor.execute(this, nodeConfiguration);
+                nodeMainExecutor.execute(this, nodeConfiguration);
 	}
 
 	private void stopRosNode() {
-		serviceClient = null;
+                serviceClient = null;
```

Comment:
Identation error inserted here.

Reply:
This seems to be a bug in GitHub PR system ... if you see in https://github.com/CST-Group/cst/blob/gudwin/src/main/java/br/unicamp/cst/bindings/rosjava/RosServiceClientCodelet.java there is no such indentation error. This was corrected in one of the commits, but for some reason it is still being reported in the PR.  Very strange !

Reply:
OK!